integration-test/src/main/java/methods/AssertionMethods.java

```diff
@@ -0,0 +1,385 @@
+package methods;
+
+import java.util.List;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+
+import env.BaseTest;
+
+public class AssertionMethods extends SelectElementByType implements BaseTest {
+    // This file contains assertion methods which are called from
+    // predefinedStepDefinitions
+
+    // SelectElementByType eleType= new SelectElementByType();
+    private WebElement element = null;
+
+    /**
+     * method to check javascript pop-up alert text
+     *
+     * @param text
+     *            : String : Text to verify in Alert
+     * @throws TestCaseFailed
+     */
+    public void checkAlertText(String text) throws TestCaseFailed {
+        if (!getAlertText().equals(text)) {
+            throw new TestCaseFailed("Text on alert pop up not matched");
+        }
+    }
+
+    /**
+     * method to check attribute value
+     *
+     * @param accessType
+     *            : String : Locator type (id, name, class, xpath, css)
+     * @param attributeName
+     *            : String : attribute name
+     * @param attributeValue
+     *            : String : attribute value
+     * @param accessName
+     *            : String : Locator value
+     * @param testCase
+     *            : Boolean : test case [true or false]
+     */
+    public void checkElementAttribute(String accessType, String attributeName, String attributeValue, String accessName,
+            boolean testCase) throws TestCaseFailed {
+        final String attrVal = getElementAttribute(accessType, accessName, attributeName);
+        if (testCase) {
```

Comment:
Amongst the following helpers, I don't understand the purpose of testCase boolean. Look like a wrapper around driver command. What is it for ? If usefull, there are maybe other way to write it.